docs/fub_curta.md

```diff
@@ -0,0 +1,33 @@
+# nf-core/configs: Freie Universiät Berlin High-Performance Computer (Curta) Configuration
+
+> **NB:** In order to run pipelines using this HPC cluster, you must first apply for access [here](https://ssl2.cms.fu-berlin.de/fu-berlin/en/sites/high-performance-computing/PM_Zugang-beantragen/index.html).
+
+This profile is configured to run with Singularity version `1.1.9-1.el7`, which does not need to be loaded.
```

Comment:
What dou mean by 'does not need to be loaded'? As in it's globally avaialble on the cluster to all users already (without any e.g. `module load` stuff)?

Reply:
Yes that's exactly what I meant. Perhaps I should word it better to make it clearer 👍🏼